mslib/msui/tableview.py

```diff
@@ -230,8 +229,10 @@ def is_roundtrip_possible(self):
             first_waypoint = self.waypoints_model.waypoint_data(0)
             last_waypoint = self.waypoints_model.waypoint_data(self.waypoints_model.rowCount() - 1)
 
-            condition = first_waypoint.lat != last_waypoint.lat or first_waypoint.lon != last_waypoint.lon or \
```

Comment:
This seems to be unrelated to the 1.3.0 activity. Can this be separated into another merge request?

Reply:
Yes. I removed it from this PR